iwyu_tool.py

```diff
@@ -435,12 +435,6 @@ def partition_args(argv):
             return argv, []
     argv, iwyu_args = partition_args(sys.argv[1:])
```

Comment:
It would be nice to rename `iwyu_args` to `extra_args` now that they're not strictly -Xiwyu args.

But please make that refactor a separate commit, as it touches several places in this file.

Reply:
Done ✔️ 
I like the suggestion. Ii've applied this into the whole iwyu_tool.